src/routes/StackView/StackView.tsx

```diff
@@ -32,58 +53,156 @@ export const StackView = () => {
-          onClose={handleClose}
-          onDeleteOfParent={handleClose}
-          showUnstackButton={false}
-          viewer={viewer}
-          className="stack-view__parent-note"
-        />
-        <NoteDialogComponents.Wrapper>
-          {stackedNotes.map((n) => (
-            <NoteDialogComponents.Note
-              key={n.id}
-              noteId={n.id}
-              text={n.text}
-              authorId={n.author}
-              avatar={n.avatar}
-              authorName={n.authorName}
-              showAuthors={showAuthors}
-              onClose={handleClose}
-              onDeleteOfParent={handleClose}
-              showUnstackButton
-              viewer={viewer}
-            />
-          ))}
-        </NoteDialogComponents.Wrapper>
+        <StackNavigation {...navigationProps} />
+        <div className="stack-view__content">
+          <Transition {...transitionConfig}>
+            {(styles: object, item: {parent: Note | undefined; stack: StackedNote[]; avatar: AvataaarProps | undefined; authorName: string}) => (
+              <animated.div style={styles} className="stack-view__animation-wrapper">
+                {item.parent?.position.column === column!.id && (
+                  <>
+                    <NoteDialogComponents.Note
+                      key={item.parent!.id}
+                      noteId={item.parent!.id}
+                      text={item.parent!.text}
+                      authorId={item.parent!.author}
+                      avatar={item.avatar}
+                      authorName={item.authorName}
+                      showAuthors={showAuthors}
+                      onClose={handleClose}
+                      onDeleteOfParent={handleClose}
+                      showUnstackButton={false}
+                      viewer={viewer}
+                      className="stack-view__parent-note"
+                    />
+                    <NoteDialogComponents.Wrapper>
+                      {item.stack?.map((n: StackedNote) => (
+                        <NoteDialogComponents.Note
+                          key={n.id}
+                          noteId={n.id}
+                          text={n.text}
+                          authorId={n.author}
+                          avatar={n.avatar}
+                          authorName={n.authorName}
+                          showAuthors={showAuthors}
+                          onClose={handleClose}
+                          onDeleteOfParent={handleClose}
+                          showUnstackButton
+                          viewer={viewer}
+                        />
+                      ))}
+                    </NoteDialogComponents.Wrapper>
+                  </>
+                )}
+              </animated.div>
+            )}
+          </Transition>
+        </div>
       </div>
+      <div className={classNames("stack-view__border", presenting && "stack-view__border--presenting", getColorClassName(column!.color as Color))} />
```

Comment:
```suggestion
      <div className={classNames("stack-view__border", {"stack-view__border--presenting": presenting}, getColorClassName(column!.color as Color))} />
```

Reply:
is there a benefit to this method if there's only one css class in the object?

Reply:
Not at this point but if there are more conditionals it will increase readability. Ignore it if you want :)